### PR TITLE
Roman

### PR DESCRIPTION
**Pull request by sodabarauf**
this is second project of javascript Roman Numeral Converter, here i did these things:
-followed GITHUB workflow.
-added linters.
-added descriptive README.md file
-deploy the link.
-added index.html, style.css, javascript.js file.
-added html tags like p, div, input, button.
-added some styles and properties to style.css file.
-added some concept of javascript.
-created asset folder for images.

### CODE REVIEW

**Comment by humuyar** [comment]
hi@sodabarauf
Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!
bravo🥳
Status Approved 👍 Great Design Implemented
✔ Created a professional Repo (You created a repository with your project name)
✔ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔ Commit messages are meaningful
✔ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔ No linter errors (All linters checks have been passed 🟢)
✔ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔Your code is clean
✔Your repo has a professional README.md file

**Comment by sodabarauf** [comment]
> hi@sodabarauf Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!! bravo🥳 Status Approved 👍 Great Design Implemented ✔ Created a professional Repo (You created a repository with your project name) ✔ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch) ✔ Commit messages are meaningful ✔ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code) ✔ No linter errors (All linters checks have been passed 🟢) ✔ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review) ✔ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request) ✔Your code is clean ✔Your repo has a professional README.md file

@humuyar 
dear humaira, tnx from your feedback.

File: styles.css
Comment (ahzamir) [inline styles.css:1]: - Although your design is impressive but it is not responsive make it responsive and request for a re-review.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image


# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️Your repo have been reviewed by your partner
❌Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: styles.css
Comment (sodabarauf) [reply styles.css:1]: > * Although your design is impressive but it is not responsive make it responsive and request for a re-review.

@ahzamir,
sir thanks from you, i make it responsive please re-review this design.

**Comment by sodabarauf** [comment]
@ahzamir 
sir, thanks from you, i make page responsive you can re-review this design.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@sodabarauf 

- Still your design is not responsive, please does not submit your project with the same mistake and issue before checking it.

image

**Comment by sodabarauf** [comment]
@ahzamir 
sir, i make it responsive form mobile versions, please re-review.
thank you.

**Review by ahzamir — APPROVED** [review]
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️Your repo have been reviewed by your partner
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by sodabarauf** [comment]
@ahzamir 
thank you much more, sir.

**Pull request merged**